examples/inferring-isolated/inferring_isolated.description

```diff
@@ -1,10 +1,10 @@
 // `isolated` is a complex feature, which would be a lot for an application developer to understand.
 // A typical Ballerina application consists of a single module that imports multiple library modules.
-// Within a single module, we can infer `isolated` qualifiers. Object w/o mutable fields is
+// Within a single module, you can infer `isolated` qualifiers. An object without mutable fields is
 // inherently `isolated`. Application developer's responsibility is to use `lock` statement where
 // needed. e.g.,
 // <ul>
-// <li>access self in a `service` object with mutable state</li>
+// <li>access `self` in a `service` object with mutable state</li>
 // <li>access mutable module-level variables</li>
```

Comment:
Shouldn't this be something like "It is the application developer's responsibility to determine when to use a `lock` statement. e.g., accessing <scenario-goes-here>"

@MaryamZi

Reply:
How about

It is the application developer's responsibility is to use `lock` statements where needed. e.g.,
....